src/Compilers/CSharp/Portable/Symbols/Tuples/TupleTypeSymbol.cs

```diff
@@ -618,6 +618,11 @@ public override NamedTypeSymbol TupleUnderlyingType
             }
         }
 
+        internal override TypeSymbol ExtensionClassTypeNoUseSiteDiagnostics
+        {
+            get { return _underlyingType.ExtensionClassTypeNoUseSiteDiagnostics; }
```

Comment:
Very likely not. However, I believe that it is still possible (if I understand correctly, we are arbitrarily binding to any ValueTuple a consumer provides, instead of a well-known one?), and crashing is not the best behavior there. (There will most likely be errors elsewhere, e.g. having an instance of a static (extension) class).